themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,104 @@
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-07-14
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test. 
+# Max length is 160 characters.
+meta_desc: Pulumi Service, the easiest way to use Pulumi’s open-source universal infrastructure as code, just got better with a new and improved user experience.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - anita-trimbur
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Earlier this year, Pulumi's user experience (UX) team sat down with users to understand how we can better serve the community. It's important not only to the UX team, but across the company, that we listen to the real wants, needs, pains, and excitements of the people who use our product.
+
+The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) was at the center of many conversations we had with users. The Pulumi Service is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. The team is always exploring how we can deliver the best SaaS experience we can. But having input from the community gave us a meaningful point of entry.
+
+<!--more-->
+
+Here are the two problems our users faced that we wanted to improve:
+
+1. Users were losing time trying to get where they needed to go. Important content needs to be quick and easy to find.
+2. Some users didn't feel like they understood the "big picture" of their operations. We wanted to offer the proper sight lines into an organization's IaC.
+
+We knew we could do better. Now, we're excited to share the changes we've made so far!
+
+## New primary navigation pane
+
+Next time you sign in to the Pulumi Service, look for the navigation pane on the left hand side of the page. All primary navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.
```

Comment:
"primary navigation items" sounds very UX-y. recommend writing it so it's oriented to the user such as "Now you can easily navigate to common selections like the Dashboard, Stacks, and Settings..."

Reply:
We've been having an open conversation about how to refer to this new navigation in public-facing writing. "Primary navigation pane" is a placeholder I've been using, but we've also talked about "left navigation," "side navigation," "vertical navigation," etc. Do you have any thoughts on the most accessible term to use?

Reply:
"Easily navigate across the Pulumi Service with new Left Navigation"?